demisto_sdk/commands/validate/validators/ST_validators/ST110_is_valid_scheme.py

```diff
@@ -5,15 +5,23 @@
     CorrelationRule,
     Dashboard,
     GenericDefinition,
+    GenericField,
+    GenericModule,
     GenericType,
     IncidentField,
     IncidentType,
     IndicatorField,
+    Job,
     Layout,
     LayoutRule,
+    Mapper,
+    ModelingRule,
+    ParsingRule,
     Playbook,
+    PreProcessRule,
 )
 from demisto_sdk.commands.content_graph.objects.integration import Integration
+from demisto_sdk.commands.content_graph.objects.list import List as List_Graph_Object
```

Comment:
```suggestion
from demisto_sdk.commands.content_graph.objects.list import List as ListObject
```
consistent with other similar imports